zerver/lib/topic.py

```diff
@@ -65,6 +67,7 @@ def REQ_topic() -> Optional[str]:
         default=None,
     )
 
+
```

Comment:
Please remove these whitespace changes inserted by your editor; our Python style is to only have one blank line between functions.